tools/clang/test/SemaHLSL/rayquery-trace-ray-inline-errors.hlsl

```diff
@@ -0,0 +1,20 @@
+// RUN: %dxc -T vs_6_5 -E main -verify %s
+
+// Test that at the call site of any TraceRayInline call, a default error
+// warning is emitted that indicates the ray query object has the
+// RAY_FLAG_FORCE_OMM_2_STATE set, but doesn't have 
+// RAYQUERY_FLAG_ALLOW_OPACITY_MICROMAPS set
+
+RaytracingAccelerationStructure RTAS;
+void main(RayDesc rayDesc : RAYDESC) : OUT {
+  // expected-note@+2{{RayQueryFlags declared here}}
+  // expected-note@+1{{RayQueryFlags declared here}}
+  RayQuery<0> rayQuery; // implicitly, the second arg is 0.
```

Comment:
Need another that shows a use of non-zero RayQueryFlags requires SM 6.9, without use of the `RAYQUERY_FLAG_ALLOW_OPACITY_MICROMAPS` identifier:
```hlsl
  RayQuery<0, 1> rayQueryInvalid;
```